caddyhttp/httpserver/server.go

```diff
@@ -490,6 +490,20 @@ func DefaultErrorFunc(w http.ResponseWriter, r *http.Request, status int) {
 	WriteTextResponse(w, status, fmt.Sprintf("%d %s\n", status, http.StatusText(status)))
 }
 
+const errStatusMisdirectedRequest = 421 // RFC 7540, 9.1.2
```

Comment:
This isn't technically an `error` value, so maybe renaming this const to `httpStatusMisdirectedRequest` would be more fitting.